solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -9,12 +9,20 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 library MinimumStakeSchedule {
     using SafeMath for uint256;
 
+    // Apr-28-2020 02:52:46 AM UTC when KEEP token has been deployed
+    // TX:  0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014
+    uint256 public constant scheduleStart = 1588042366;
```

Comment:
Why not define this value in `TokenStaking` contract? The value could be provided in the `TokenStaking` constructor.

Reply:
Oh size limitation, huh?

Reply:
Yes. Also, we can save on gas when this value is not passed as a parameter. It never changes and `MinimumStakeSchedule` is a part of `TokenStaking`.

Reply:
I don't like hardcoding this value, but we have to 🤷